packages/core/src/components/Accordion.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import {
+  View,
+  StyleSheet,
+  StyleProp,
+  TextStyle,
+  Pressable,
+} from "react-native";
+import Text from "./Text";
+import { withTheme } from "../theming";
+import type { IconSlot } from "../interfaces/Icon";
+import type { Theme } from "../styles/DefaultTheme";
+import { colorTypes } from "@draftbit/types";
+
+import {
+  COMPONENT_TYPES,
+  createTextProp,
+  createNumberProp,
+  createIconProp,
+  createColorProp,
+} from "@draftbit/types";
+
+type Props = {
+  openColor: colorTypes;
+  closedCloser: colorTypes;
+  caretColor: colorTypes;
+  icon?: string;
+  iconSize: number;
+  style: StyleProp<TextStyle>;
+  children: React.ReactNode;
+  title: string;
+  theme: Theme;
+} & IconSlot;
+
+const Accordion = ({
+  Icon,
+  openColor,
+  closedCloser,
+  //caretColor,
+  icon,
+  iconSize = 24,
+  style,
+  title,
+  children,
+  theme: { colors },
+}: Props) => {
+  const [expanded, setExpanded] = React.useState(false);
+
+  const handlePressAction = () => {
+    setExpanded(!expanded);
+  };
+
+  const titleColor = expanded ? colors[openColor] : colors[closedCloser];
+
+  return (
+    <View>
+      <Pressable
+        style={[styles.container, style]}
+        onPress={handlePressAction}
+        accessibilityRole="button"
+      >
+        <View style={styles.row} pointerEvents="none">
```

Comment:
what does pointerEvents none do here?

Reply:
Nothing actually, it was in the original file hence included it, removing it as it not recommended anymore in the docs.